app/src/main/java/com/poupa/vinylmusicplayer/helper/SearchQueryHelper.java

```diff
@@ -1,92 +1,59 @@
 package com.poupa.vinylmusicplayer.helper;
 
 import android.app.SearchManager;
-import android.content.Context;
 import android.os.Bundle;
 import android.provider.MediaStore;
 
 import androidx.annotation.NonNull;
 
+import com.poupa.vinylmusicplayer.discog.Discography;
+import com.poupa.vinylmusicplayer.util.StringUtil;
 import com.poupa.vinylmusicplayer.loader.SongLoader;
 import com.poupa.vinylmusicplayer.model.Song;
 
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiPredicate;
+import java.util.function.Function;
+import java.util.function.Predicate;
 
 /**
  * @author Karim Abou Zeid (kabouzeid)
+ * @author SC (soncaokim)
  */
 public class SearchQueryHelper {
-    private static final String TITLE_SELECTION = "lower(" + MediaStore.Audio.AudioColumns.TITLE + ") = ?";
```

Comment:
Completely rewritten - I dont know how this can be tested though (is it related to Android Auto? I dont have a working setup to emulate this)